devicon.json

```
@@ -1,4 +1,29 @@
 [
+    {
+        "name": "aerospike",
+        "altnames": [
+            "aerospike"
+        ],
+        "tags": [
+            "data",
+            "database",
+            "nosql"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain"
+            ],
+            "font": [
+                "original",
+                "original-wordmark",
+                "plain"
+            ]
+        },
+        "color": "#C31618",
+        "aliases": []
```

The `original.svg` is simple enough (only has 1 path and fill) allowing to be used as an alias. 

```suggestion
        "aliases": [
            {
               "base": "original",
               "alias": "plain"
            }
        ]
```